test/gzip_specials.js

```diff
@@ -86,4 +86,17 @@ describe('Gzip special cases', function () {
     assert.deepEqual(inflator.result, expectedDataArray, 'inflator produced expected data');
   });
 
+  it('Read bgzipped file 1', function () {
```

Comment:
It will be more convenient if all 3 tests will have similar names and fixtures:

- "Read streamed file 1" + "streamed_gzip_java.txt.gz
- "Read streamed file 2" + "streamed_bgzip_1.txt.gz
- "Read streamed file 3" + "streamed_bgzip_2.txt.gz